cranelift-wasm/tests/wasm_testsuite.rs

```diff
@@ -45,6 +45,35 @@ fn use_fallthrough_return() {
     );
 }
 
+#[test]
+fn use_name_section() {
```

Comment:
Added a test to check that the name section is parsed, and that module/functions/local names are declared. This meant adding a small test fixture, `wasmtests/names.wat`